pyvista/core/filters/poly_data.py

```diff
@@ -2086,7 +2098,7 @@ def multi_ray_trace(self, origins, directions, first_point=False, retry=False):
 
         """
         if not self.is_all_triangles:
```

Comment:
```suggestion
        if not self.is_all_triangles:  # pragma: no cover
```

Reply:
I don't think this is the right call. I added a test for `multi_ray_trace` for this branch. The issue is that coverage probably doesn't have the optional dependencies needed for that test, so it skips it. But this relates to the entire `multi_ray_trace` functionality, not just the diff. If we want to skip this for coverage, we should skip the whole method @akaszynski.

Reply:
Fixed in 051122a780ccd14fc4f547839ae28f71c360338b.